### DISCUSSION TITLE
Anyone selling a boards, kits or complete units

### DISCUSSION THREAD

**Discussion by TurboOverLord** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#496] [General]
Hi all

Anyone selling a board, kits or a complete unit ?
I am from the UK and happy to cover all postage costs from wherever you are located.

Many thanks Lee

**Comment by dasfliege** [comment]
I have a bunch of plain v2 boards, but i'm located in switzerland. Guess it's faster and cheaper if you just order 5 pieces at JLCPCB

**Reply by reevery** [reply]
@dasfliege Do you still have any boards? I am also in Switzerland (Zurich).

**Comment by wyliecoyoteuk** [comment]
I have a couple of v1 boards spare and I am in the UK

**Comment by TurboOverLord** [comment]
Hi

Can you email me please allroundautos@gmail.com

Many thanks lee

**Reply by challo2018** [reply]
Did you manage to get PCB's? I have some left, located at Germany

**Reply by Remigiusz2020** [reply]
hi, Im from Poland. What type pcb do you have ?

**Reply by challo2018** [reply]
I have latest version V2B

**Reply by Techtobi83** [reply]
Hi Challo2018, i am from germany and are interested to buy 1 pcb from your stock, if still available? 
How can i contact you?

**Reply by armageddon421** [reply]
@challo2018 hey, I'm also located in Germany and interested in one or two boards. My mail is visible in my profile. I'll be happy to hear from you!

**Comment by TurboOverLord** [comment]
Hi

Thank you for your reply
What type are they ?

Regards lee

**Comment by TurboOverLord** [comment]
Hello

Thank you for your email.
Are you able to supply a built unit ?

Regards lee

**Comment by TurboOverLord** [comment]
Hello

Thank you for your response.
I have the maldives with hydrojet so I'm guessing I'd need the
latest version.  But I've not checked the pump pcb yet.

Regards lee

**Comment by Remigiusz2020** [comment]
ok. Im from Poland. Can you post it to me?  How much for v2 and delivery ? 
            
          
     
      
       
        Dnia 19 czerwca 2023 23:27 Fabian ***@***.***&gt; napisał(a):
       
    
       
         
 I have latest version V2B

**Comment by Remigiusz2020** [comment]
Hi. I dont have to sell. I want to buy v2b.    Dnia 24 czerwca 2023 11:26 Techtobi83 ***@***.***&gt; napisał(a):   Hi Challo2018, i am from germany and are interested to buy 1 pcb from your stock, if still available?   How can i contact you?  —  Reply to this email directly,  github.com view it on GitHub , or  github.com unsubscribe .  You are receiving this because you commented. Message ID:  &lt;visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/repo-discussions/496/comments/6268341 @ github . com&gt;

**Comment by bartbakels** [comment]
I am also looking for a pcb v2 if possible my selfbuild one has some issues lately. Please contact me on bartbakels@ziggo.nl located in NL

**Comment by AndyMu97** [comment]
I do have 1 complete V2 set to sell in Germany incl. 
- LoLin nodemcu V3 (from az-delivery.de)
- Red & Blue LLC
- Cabling
- Female headers
![IMG_0425](https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/assets/138725961/dba5b145-d096-4883-a86a-58703bb6994b)

**Reply by reevery** [reply]
Do you still have a kit available? How much including for postage to DE?

**Reply by AndyMu97** [reply]
@reevery how can I get in direct contact with you? Don’t want to post my mail address.

**Reply by AndyMu97** [reply]
@reevery already sent you a mail, did you see?

**Comment by bartbakels** [comment]
Got one already thanksOp 11 jul. 2023 om 20:59 heeft AndyMu97 ***@***.***> het volgende geschreven:﻿
I do have 1 complete V2B set to sell in Germany incl.

LoLin nodemcu V3 (from az-delivery.de)
Red & Blue LLC
Cabling
Female headers

**Comment by TurboOverLord** [comment]
Hello

How much would you want for it with postage to the UK?

Best regards lee

**Reply by AndyMu97** [reply]
Wouldn‘t make sense, minimum 15,49€ postage only with no tracking. Parcel with tracking 26,99€.
Plus the set price.

**Reply by metaljay** [reply]
https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/580

**Comment by TurboOverLord** [comment]
Thank you for your reply
That sound great.
Can I see a photo of what you have please.

Regards lee

**Comment by TurboOverLord** [comment]
Hi can you give me your PayPal details so I can send you the payment. I'd
like to go for fully tracked. Can you let me know how much the kit will be.
Thanks regards lee

**Reply by AndyMu97** [reply]
It is already sold, the parcel alone was 27 Euro only for transport, no content!
So it hasn’t made any sense when transport is higher than the value of content.

**Comment by TurboOverLord** [comment]
Sorry I'm a bit confused. You sold it to someone else?

**Reply by AndyMu97** [reply]
@TurboOverLord would be easier if you reply within one thread and not opening new ones.
My set isn’t sold as reevery found another one, but do you really want to pay >45 EUR for a set just because of the high transport costs?
And… to be honest, it is a nightmare to send international especially outside EU because of customs declarations, so I will only send national/Germany. Sorry 🙂

**Comment by Bryla2507** [comment]
I have 3 V2B boards to sell in Poland. I can ship here, maybe EU

**Reply by Eurycleia** [reply]
Do you have a complete kit available (incl cabling and such?)

**Comment by AndyMu97** [comment]
Still have one set to sell, deliver within Germany:
https://github.com/visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA/discussions/496#discussioncomment-6419706

If someone is interested, please send me an email to 
fabelhaftzuverlaessigergepard@fukaru.com

**Reply by loxar02** [reply]
You have a mail :)
